<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MATRIZ A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>multiplicação por uma K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>gerar tabela</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaMatB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>231</width>
      <height>74</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdicionar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Quantidade de Colunas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditColunasMatB"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEditSaida">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>matriz transposta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MATRIZ B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConstanteB">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>multiplicação por uma K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMatA">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>CRIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMulti">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Quantidade de Linhas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLinhasMatB"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Quantidade de Linhas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLinhasMatA"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonTranspostaB">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>matriz transposta</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Quantidade de Colunas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditColunasMatA"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMatB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>CRIAR</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetGrid">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>TRIANGULAR SUPERIOR</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>TRIANGULAR INFERIOR</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SIMÉTRICA</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>IDENTIDADE</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>IGUAIS</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>DIFERENTES</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>ORTOGONAL</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>PERMUTAÇÃO</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>MATRIZ A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MATRIZ B</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaMatA">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>231</width>
      <height>74</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
